devices/milightRF24.coffee

```diff
@@ -78,7 +92,28 @@ module.exports = (env) ->
           button = Buttons.Group4Off
       
       @gateway.sendButton(id, zone, button, false)
+      
+      @_loop(id, zone, button, false, 0, 0, 0,0,0)
+  
+    _loop: (id, zone, button, longPress, discoMode, brightness, r, g, b) ->
+      dataObj =
+        raw: "loop",
+        id: id,
+        zone: zone,
+        button: button,
+        longPress: longPress,
+        discoMode: discoMode,
+        brightness: brightness,
+        color:
+          r: r,
+          g: g,
+          b: b
+          
+      @.emit("ReceivedData", dataObj);
     
+  MilightRF24::__proto__ = events.EventEmitter.prototype
```

Comment:
Maybe
This was how I found it in an example.

I will change it and try it out.

2015-08-04 14:43 GMT+02:00 philip1986 <notifications@github.com>:

> In devices/milightRF24.coffee
> <https://github.com/philip1986/pimatic-led-light/pull/23#discussion_r36182778>
> :
>
> >
> > +  MilightRF24::__proto__ = events.EventEmitter.prototype
>
> Why you do it that way?
>
> Wouldn't class MilightRF24 extends events.EventEmitter have the same
> effect?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/philip1986/pimatic-led-light/pull/23/files#r36182778>.
>